packages/vscode/src/generated/meta.ts

```diff
@@ -4,7 +4,7 @@
 // Meta info
 export const publisher = 'antfu'
 export const name = '@unocss/vscode'
-export const version = '0.65.3'
+export const version = '65.4.0'
```

Comment:
Can you pull the latest code and resolve conflicts?

Reply:
Done. It was a commit from the CI